In [4]:
import numpy as np
import pandas as pd
import matplotlib.pyplot as plt
import scipy.stats as st
import seaborn as sns


In [2]:
Data=[
1.13, 1.55, 1.43, 0.92, 1.25, 1.36, 1.32, 0.85, 1.07, 1.48, 1.20, 1.33, 1.18, 1.22, 1.29
]
print(Data)

[1.13, 1.55, 1.43, 0.92, 1.25, 1.36, 1.32, 0.85, 1.07, 1.48, 1.2, 1.33, 1.18, 1.22, 1.29]


In [5]:
df = pd.DataFrame(Data)

In [6]:
np.std(df,ddof=1)

0    0.193164
dtype: float64

In [7]:
df.describe()

0
count  15.000000
mean    1.238667
std     0.193164
min     0.850000
25%     1.155000
50%     1.250000
75%     1.345000
max     1.550000

99% Confidence Interval Using Sample Standard Deviation


In [8]:
st.t.interval(confidence=0.99,df=len(df)-1,loc=np.mean(df),scale=st.sem(df))

(array([1.09019734]), array([1.38713599]))

The t-distribution is a way of describing a set of observations where most observations fall close to the mean, and the rest of the observations make up the tails on either side.
Frommthe above sample we observed that 99% of the Print Heads fails between 1.0901.... and 1.3871... .

99% Confidence Interval Using Known Population Standard Deviation

In [9]:
st.t.interval(confidence=0.99,df=len(df)-1,loc=np.mean(df),scale=0.2)

(0.6432981198441345, 1.8340352134891986)